### ISSUE TITLE
Problems pane completely empty despite `state` having stuff under `state.problems`

### ISSUE THREAD

**Issue by maoueh** [FormidableLabs/electron-webpack-dashboard#56]
While working on #55, on current master commit (4886479), I noticed that even if `state.problems` has something set inside it, the UI is still not displaying anything in the pane.

**Comment by maoueh** [comment]
Seems to be a rendering problem with `flex: 0`:

```
File: app/containers/problems.js
15: const ProblemContainer = withSettings(styled(ScrollerBase)`
16: > flex: 0;
17:   white-space: pre-wrap;
18:   font-family: 'menloregular';
19:   font-size: ${({ fontSizeModifier }) => getFontSize(11, fontSizeModifier)}px;
20:   color: #fff;
21:   opacity: 0;
22:   animation: 500ms ${fadeInAnimation} 500ms;
23:   animation-fill-mode: forwards;
24:   height: calc(100% - 27px);
25: `);
26: 
```

Changing from `flex: 0` to `flex: 1` fixed the problem. Not sure if it's a good fix or not, I leave you the call on this one.

**Comment by mattcubitt** [comment]
Fixed in #57
